server/src/test/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessorTest.java

```diff
@@ -273,6 +274,112 @@ public void testSeqAndUnSeqSyncClose()
     }
   }
 
+  @Test
+  public void testEnableDiscardOutOfOrderDataForInsertRowPlan()
+      throws WriteProcessException, QueryProcessException, IllegalPathException {
+    IoTDBConfig config = IoTDBDescriptor.getInstance().getConfig();
+    config.setEnableDiscardOutOfOrderData(true);
```

Comment:
remember to reset this to its default value in the end of the test

We usually do this as follows:

boolean defaultValue = config.isEnableDiscard...;
config.setEnableDiscard(true);
do some test
config.setEnableDiscard(defaultValue);